cmd/kured/main.go

```diff
@@ -205,6 +214,19 @@ func flagCheck(cmd *cobra.Command, args []string) {
 			notifyURL = fmt.Sprintf("slack://%s", strings.Trim(trataURL.Path, "/services/"))
 		}
 	}
+	var preRebootNodeLabelKeys []string
```

Comment:
We can combine these two variable declarations into a single line, e.g.:

```
var preRebootNodeLabelKeys, afterRebootNodeLabelKeys []string
```

Reply:
OK, will do.